qrenderdoc/Windows/Dialogs/RemoteManager.cpp

```diff
@@ -198,7 +198,10 @@ void RemoteManager::runRemoteServer(RDTreeWidgetItem *node)
   if(!host)
     return;
 
-  host->Launch();
+  if(IsHostADB(host->Hostname))
+    RENDERDOC_StartAndroidRemoteServer();
+  else
+    host->Launch();
```

Comment:
When testing this I saw that although this works fine if you click run remote server in the remote manager dialog, that's not the common path - I think this code should just be moved inside `host->Launch()` because then it will act like any other run command and automatically launch the server when people switch to the android replay context.